app/Http/Controllers/EpisodeManager.php

```diff
@@ -23,9 +33,21 @@ class EpisodeManager extends Controller
     */
     public function index()
     {
-        $episodes = Episode::all();
-
-        return view('dashboard/pages/view_episodes', compact('episodes'));
+        $data = [
+            "user"      =>  [  "total"     => $this->userRepository->getAllUser(),
+                                "online"    => $this->userRepository->getOnlineUsers()->count(),
+                                "offline"   => $this->userRepository->getOfflineUsers()->count() 
+                            ],
+            
+            "episodes" =>   [
+                                "recent"    => $this->episodeRepository->getAllEpisode(),
+                                "active"    => $this->episodeRepository->getActiveEpisode(),
+                                "pending"   => $this->episodeRepository->getPendingEpisode()
+                            ],
+
+            "channels"      => $this->channelRepository->getAllChannel()
+        ];
+        return view('dashboard/pages/index', compact('data'));
```

Comment:
There should be a newline before `return`. I have said this more than a million times.

I said that we should adopt using this style `dashboard.pages.index` rather than `dashboard/pages/index`.

Please pay attention to details and take correction